nc-news-project/src/components/Home.js

```diff
@@ -36,10 +38,11 @@ const Home = () => {
             }
         }))
     })
+    setSortList([{value: 'date', label: 'Date: Oldest to newest'}, {value: 'votes&order=desc', label:'Votes: Higest'}, {value: 'votes&order=asc', label: 'Votes: Lowest'}, {value: 'author&order=asc', label: 'Author: A-Z'}, {value: 'author', label: 'Author: Z-A'}])
```

Comment:
Theres no real need for this to be a piece of state since you are not updating it from the page, you could just have this as a normal static variable on in the component